packages/react-native-renderer/src/findNodeHandle.js

```diff
@@ -98,24 +87,11 @@ function findNodeHandle(componentOrHandle: any): any {
   // ReactInstanceMap.get here will always succeed for mounted components
   const internalInstance: Fiber = ReactInstanceMap.get(component);
   if (internalInstance) {
-    return (
-      findHostInstance(internalInstance) ||
-      findHostInstanceFabric(internalInstance)
-    );
+    return findHostInstance(internalInstance);
   } else {
     if (component) {
       return component;
     } else {
-      invariant(
-        // Native
-        (typeof component === 'object' && '_nativeTag' in component) ||
-          // Composite
-          (component.render != null && typeof component.render === 'function'),
```

Comment:
This is always falsy according to the branch above. So never can pass. This must throw?

Reply:
lol. brb going to add a function to Number.prototype.render